documentation/book/proc-configuring-clients-to-trust-cluster-ca.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// assembly-security.adoc
+
+[id='configuring-clients-to-trust-cluster-ca-{context}']
+= Configuring clients to trust the Cluster CA 
+
+If a Kafka client wants to connect to the `tls` listener on port 9093 or the `external` listener on port 9094, it needs to trust the cluster CA certificate.
+
+.Procedure
+
+. The cluster CA certificate can be extracted from the generated `_<cluster-name>_-cluster-ca-cert` `Secret`.
++
+ifdef::Kubernetes[]
+On {KubernetesName}, the certificate can be extracted with the following command:
++
+[source,shell]
```

Comment:
You need subs="+quotes" in your [source] block for replaceables to render properly - also you need to remove 'shell' - for some reason that causes errors.  All of this here: http://strimzi.io/contributing/guide/#replaceables

Reply:
It renders fine for me even with the `shell`. It would be rubbish if some of these things were rendered as shell and others not simply on the basis of whether they contained italics. So if it's really a problem having `shell` with `subs="+quotes"` maybe we should consider removing `shell` everywhere.

Reply:
This is a strange one - I tried with several permutations and the only way I could get the doc to build right downstream was without 'shell' - I agree with you on the fact that it's 'rubbish' :)  This is something maybe @dmuntima and @laidan6000 can investigate more.